rclcpp/include/rclcpp/node.hpp

```diff
@@ -101,6 +101,12 @@ class Node : public std::enable_shared_from_this<Node>
   const char *
   get_name() const;
 
+  /// Get the namespace of the node.
+  /** \return The namespace of the node. */
+  RCLCPP_PUBLIC
+  const char *
+  get_namespace() const;
```

Comment:
Mabe add a simple test in this package too, since this would have been caught by that.